intranet/static/css/page_base.scss

```diff
@@ -43,6 +43,7 @@ h1 {
     .right {
         -webkit-user-select: none;
         user-select: none;
+        padding-right:20px;
```

Comment:
I would prefer to have this as an inline style on the `<div>`, since we may have other uses of the `right` class that do not need padding.